AUTHORING_GUIDELINES.md

```diff
@@ -0,0 +1,14 @@
+# Authoring guidelines
+
+We want to make it easier to write awesome CSS that works great in as many browsers as possible.
+Some of these browsers are a bit older, others don't yet exist.
+
+We will not place undue burden on anyone writing or maintaining CSS.
+
+We will only add a plugin to `postcss-preset-env` when the CSS WG invites implementations for the relevant features.
+
+We will not create plugins with features that look and feel native but don't have a specification.
```

Comment:
```suggestion
We will not create plugins with features (or add features to plugins) that look and feel native but don't have a specification.
```

Feel free to tweak the wording, an example would be #671

Reply:
I agree, good catch :)

It is something we have seen in the past.
That small productivity features find their way into plugins that intended to "polyfill" native features.

Best to keep those separate.